pkgs/by-name/tr/tritty/package.nix

```
@@ -0,0 +1,43 @@
+{ callPackage, fetchFromGitHub, lib, stdenv }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tritty";
+  # The number after + indicates the number of commits since the release.
+  version = "1.0+12";
```

After reading about [Package naming](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming) in the nixpkgs README.md I wonder whether the version for 
I'd like to suggest to use the following version, which would follow the recommendations given in the section [**Package naming**](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming) of the nixpkgs README.md.

What are your thoughts on this, @roberth?

```suggestion
  version = "1.0-unstable-2018-12-06";
```